### DISCUSSION TITLE
Show Langs Card not showing Java language

### DISCUSSION THREAD

**Discussion by rubanero14** [anuraghazra/github-readme-stats#3691] [Q&A] [answered]
### Describe the bug

Despite my profile 3rd highest use of Java language not showing in top 10 languages in this particular Langs card. 

### Expected behavior

Expected to show Java as one of the top 3 language in profile, but omitted out. Please look into code base and check if the algorithm takes into account Java param

### Screenshots / Live demo link

My GitHub profile [link](https://GitHub.com/rubanero14)

**Comment by MagdielPaulo** [comment]
Check your Langs card settings and make sure they are configured to show the languages ​​you want to see. Check out the GitHub documentation to learn more about how the Langs card works and research Java's popularity on GitHub and other development platforms.

**Comment by qwerty541** [comment, marked as answer]
Hey, @rubanero14! Currently top languages card shows data only about first 100 repositories (https://github.com/anuraghazra/github-readme-stats/blob/master/src/fetchers/top-languages-fetcher.js#L30). This is because GitHub API limitations which cause downtimes of public instances. In future this behavior will be improved by releasing GitHub action or providing environment variables for user's own instances. Our documentation contains that information.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/cefd6524-2677-4a83-abaf-63dfecca872f)
